### PR TITLE
Sys 328 fix warnings during compile time 3.3.0

### PR DESCRIPTION
**Pull request by oshchepkov**
Fixed big portion of compile warnings. The ones that remain unfixed are mostly related to signed/unsigned comparison and require some degree of a refactoring.

Tests pass:
image

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -101,25 +101,25 @@ static CBlock CreateGenesisBlock(uint32_t nTime, uint32_t nNonce, uint32_t nBits
...
-}
+// static void GenerateGenesisBlock(CBlockHeader &genesisBlock, uint256 &phash)
+// {
+// 	arith_uint256 bnTarget;
+// 	bnTarget.SetCompact(genesisBlock.nBits);
+// 	uint32_t nOnce = 0;
+// 	while (true) {
+// 		genesisBlock.nNonce = nOnce;
+// 		uint256 hash = genesisBlock.GetHash();
+// 		if (UintToArith256(hash) <= bnTarget) {
```
Comment (jnessbcf) [inline src/chainparams.cpp]: Delete commented code?

**Review by jnessbcf — CHANGES_REQUESTED** [review]
Minor feedback

File: src/chainparams.cpp (hunk 1 above)
Comment (oshchepkov) [reply src/chainparams.cpp]: @jnessbcf, ok

**Review by jnessbcf — APPROVED** [review]
utACK

**Pull request merged**